src/main/java/com/cdancy/jenkins/rest/JenkinsClient.java

```diff
@@ -121,6 +134,7 @@ public static Builder builder() {
         private String endPoint;
         private JenkinsAuthentication.Builder authBuilder;
         private Properties overrides;
+        private List<Module> modules;
```

Comment:
Taking into account the below comment we could initialize this to an empty list here.